.github/workflows/spec.yml

```diff
@@ -0,0 +1,17 @@
+name: Spec
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    container:
+      image: crystallang/crystal:1.0.0
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: shards install
+    - name: Run crystal spec
+      run: crystal spec
```

Comment:
Two questions on this, since I've never used GH Actions:

1. Is there anything special we need to do at the repo level to get this working?
2. Is there an easy way to run this against _both_ 1.0.0 and [1.1.0](https://hub.docker.com/layers/crystallang/crystal/1.1.0/images/sha256-09ac7dc17b37759cb575aac4704e70be67de3908ddebf49c1bd4b858d81f6a07?context=explore) released this week?

Reply:
> Is there anything special we need to do at the repo level to get this working?

Nope, it should just work™ as soon as it's merged.
(e.g. see the little green checkmark on the [origin repo)[https://github.com/m-o-e/protobuf.cr/tree/fix-ci])

> Is there an easy way to run this against both 1.0.0 and 1.1.0 released this week?

Ah, good call!
Yup, added it to the yaml.

Reply:
Fantastic! Thanks!